src/index.html

```diff
@@ -13,12 +13,14 @@
     </head>
     <body>
         <nav class="navigation">
+            <img src="/src/img/Logo.png" class="navigation__logo" alt="Doctor Care logo">
             <ul class="navigation__list">
                 <li class="list__item"><a href="#">Início</a></li>
                 <li class="list__item"><a href="#">Sobre</a></li>
                 <li class="list__item"><a href="#">Serviços</a></li>
                 <li class="list__item"><a href="#">Depoimentos</a></li>
             </ul>
```

Comment:
why is there a button inside the navigation? it should be part of the header and not the navigation. Fix that and pay more attention to your implementation.